style.css

```diff
@@ -0,0 +1,455 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Kanit', sans-serif;
+}
+
+.close {
+  text-align: start;
+  width: 100vw;
+  height: 100vh;
+  padding-left: 7.5%;
+  padding-top: 5%;
+  padding-right: 10%;
+  top: 0%;
+  left: -110%;
+  color: #fff;
+  background-color: #272a31f7;
+  opacity: 98%;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  text-decoration: none;
+}
+
+.close.active {
+  left: 0%;
+}
+
+.close-list a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.close-list {
+  list-style-type: none;
+  font-size: 30px;
+  padding: 3%;
+  line-height: 10vh;
+  margin-bottom: 50px;
+}
+
+.cross {
+  text-align: right;
+  font-size: 30px;
+  cursor: pointer;
+}
+
+.navigation {
+  display: none;
+}
+
+.black-nav {
+  display: none;
+}
+
+.hamburger {
+  color: #808080;
+  background: none;
+  border: none;
+  font-size: 30px;
+  padding: 3%;
+  cursor: pointer;
+}
+
+.logo-img {
+  width: 18%;
+  display: inline-block;
+  padding: 0%;
+  margin-top: 2%;
+  margin-bottom: 2%;
+}
+
+.mobile-logo {
+  display: flex;
+  flex-direction: row;
```

Comment:
### OPTIONAL

- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)